mindsdb/integrations/handlers/gcs_handler/gcs_tables.py

```diff
@@ -9,14 +9,25 @@ class ListFilesTable(APIResource):
     def list(self,
              targets: List[str] = None,
              conditions: List[FilterCondition] = None,
+             limit: int = None,
              *args, **kwargs) -> pd.DataFrame:
 
-        tables = self.handler._get_tables()
+        buckets = None
+        for condition in conditions:
+            if condition.column == 'bucket':
+                if condition.op == FilterOperator.IN:
```

Comment:
`FilterOperator` has been used, but not imported.

Reply:
Imported